examples/miral-shell/miral-xrun.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-x11_server=Xmir
+x11_server=Xwayland
+
+if [ $(which Xmir 2>/dev/null) ]
```

Comment:
I'd just drop `[ $( ) ]` altogether then. `which` does the right checks already.